css/global.css

```diff
@@ -332,6 +335,256 @@ Global Component Styles (buttons, forms, etc.)
 	color: var(--color-onyx);
 }
 
+/* HEADER STYLES BB */
+/*
+===================================
+===================================
+Styles for an individual page
```

Comment:
Please feel free to remove these instruction comments! just leave the main heading comments for easier readability